index.html

```diff
@@ -51,5 +51,152 @@ <h4>I'm Azeezat</h4>
     </ul>
     <div id="background"></div>
     </section>
+    <div class="work-section">
+    <h1>My Recent work</h1>
+    </div>
+
+    <hr class="solid"/>
+
+    <section class="picture">
+    <div class="pictures"></div>
+    <p id="post">Multiple-Post Stories</p>
+    <p class="daily">
+        A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard dummy text ever since 1500s,
+        when an unknown printer took a dummy text.
+    </p> 
+    <div class="grid">
+    <ul class="grid-container">
+        <li class="grid-item">Css</li>
+        <li class="grid-item">Html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+    </ul>
```

Comment:
- [ ] Good job on this project so far 👍 like the previous CR mentioned the entire 'work section' should be inside a `<section>` tag and styled by using grid. please refer to [this article](https://css-tricks.com/snippets/css/complete-guide-grid/) if you're not sure or ask your question inside the PR. (I wasn't able to make an inline comment for the entire section because it's to big but make sure you make the changes for the entire work section.)